src/test/organisation/searchusers/get.controller.test.js

```diff
@@ -56,6 +57,72 @@ describe('Organisation Search Users Get Controller', () => {
     });
   });
 
+
+  it('should display error message if set', async () => {
+    req.session.errMsg = { message: 'Example Error Message' };
+      const cookie = new CookieModel(req);
+      orgApiStub.resolves(JSON.stringify({
+        items: [
+          { id: 'PERSON-1', first_name: 'PERSON-1-name', role: { name: 'User' } },
+          { id: 'PERSON-4', first_name: 'PERSON-1-name', role: { name: 'User' } },
+          { id: 'PERSON-5', first_name: 'PERSON-1-name', role: { name: 'User' } },
+        ],
+        
+      }));
+
+    const callController = async () => {
+        await controller(req, res);
+      };
+
+    callController().then(() => {
+      expect(req.session.errMsg).not.to.be.undefined;
+      expect(orgApiStub).to.have.been.calledOnceWithExactly('ORGANISATION-ID-1', 'TEST' );
+      expect(res.render).to.not.have.been.calledOnceWithExactly('/organisation/index', {
+        cookie,
+        orgUser: [
+          { id: 'PERSON-1', first_name: 'PERSON-1-name', role: { name: 'User' } },
+          { id: 'PERSON-4', first_name: 'PERSON-1-name', role: { name: 'User' } },
+          { id: 'PERSON-5', first_name: 'PERSON-3-name', role: { name: 'Admin' } },
+        ],
+        errors: [{ message: 'Example error message' }]
+      });
+      
+    });
+  });
+
+
+  it('should display success message if set', async () => {
+    const cookie = new CookieModel(req);
+
+    orgApiStub.resolves(JSON.stringify({
+      items: [
+        { id: 'PERSON-1', first_name: 'PERSON-1-name', role: { name: 'User' } },
+        { id: 'PERSON-4', first_name: 'PERSON-1-name', role: { name: 'User' } },
+        { id: 'PERSON-5', first_name: 'PERSON-3-name', role: { name: 'Admin' } },
+      ],
+    }));
+    const callController = async () => {
+      await controller(req, res);
+    };
+
+    callController().then(() => {
+      expect(req.session.successMsg).to.be.undefined;
+      expect(req.session.successHeader).to.be.undefined;
+      expect(orgApiStub).to.have.been.calledOnceWithExactly('ORGANISATION-ID-1','TEST');
+      expect(res.render).to.not.have.been.calledOnceWithExactly('/organisation/index', {
+        cookie,
+        orgUser: [
+          { id: 'PERSON-1', first_name: 'PERSON-1-name', role: { name: 'User' } },
+          { id: 'PERSON-4', first_name: 'PERSON-1-name', role: { name: 'User' } },
+          { id: 'PERSON-5', first_name: 'PERSON-3-name', role: { name: 'Admin' } },
+        ],
+        successHeader: 'Successful header',
```

Comment:
Question: Where is the `successHeader` defined in the this test?
* I can't find in the file, but I could be misunderstand thing.